homeassistant/components/bryant_evolution/climate.py

```diff
@@ -0,0 +1,212 @@
+"""Support for Bryant Evolution HVAC systems."""
+
+from datetime import timedelta
+import logging
+from typing import Any
+
+from evolutionhttp import BryantEvolutionClient
+
+from homeassistant.components.climate import (
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACAction,
+    HVACMode,
+)
+from homeassistant.const import CONF_HOST, UnitOfTemperature
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import BryantEvolutionConfigEntry
+from .const import CONF_SYSTEM_ID, CONF_ZONE_ID
+
+_LOGGER = logging.getLogger(__name__)
+
+
+SCAN_INTERVAL = timedelta(seconds=60)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: BryantEvolutionConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up a config entry."""
+    host = config_entry.data[CONF_HOST]
+    system_id = config_entry.data[CONF_SYSTEM_ID]
+    zone_id = config_entry.data[CONF_ZONE_ID]
+    client = config_entry.runtime_data
+    climate = BryantEvolutionClimate(host, system_id, zone_id, client)
```

Comment:
```suggestion
    climate = BryantEvolutionClimate(config_entry.data[CONF_HOST], config_entry.data[CONF_SYSTEM_ID], config_entry.data[CONF_ZONE_ID], config_entry.runtime_data,)
```
just preference, to not define variables we only use once anyways

Reply:
done for the config_entry.data ones. I left climate to make it clear what it is ("runtime_data" isn't an informative name).